test/artefacts/marriage-abroad/austria/ceremony_country/partner_british/same_sex.txt

```diff
@@ -1,4 +1,4 @@
-Civil partnership in Austria
+Same-sex marriage in Austria
```

Comment:
You could update the erb file the flattening script produced to avoid this content change.  I imagine this will change at some point anyway, but it means you don't need to get a content designer to approve this now.

Reply:
Thanks @barrucadu I've made the change to remove the test artefacts.